app/assets/stylesheets/components/_navigation-sidebar.scss

```diff
@@ -11,10 +11,10 @@
 .app-c-navigation-sidebar__sub-heading {
   border-top: 1px solid $border-colour;
   padding-top: $gutter-half;
-}
 
-.app-c-navigation-sidebar__external-heading {
-  @include bold-19;
+  &--bold {
```

Comment:
(non blocking) nit: gov.uk frontend is writing selectors out in full so they're easier to discover in the source code, for example when searching in github.